src/main/java/jmgomez/apipolicy/feignClient/PolicyClient.java

```diff
@@ -8,7 +8,7 @@
 
 import java.util.List;
 
-@FeignClient(name = "policyClient", url = "${url_wiremock}")
+@FeignClient(name = "policyClient", url = "${spring.application.serviceUrl}")
```

Comment:
Closer, but too generic. What if we have more than one service? Maybe we can identify this one a bit, something about policies or something like that... Also, is it the full URL? or just the base URL? With those two hints you should be able to come up with a better name